### ISSUE TITLE
Auth Laravel

### ISSUE THREAD

**Issue by hoangsang17th** [hoangsang17th/TaskE#1]
I try to change the default database of the Laravel auth method, how can I do it?

**Comment by Hamza1-coder** [comment]
Yes you do it create a database and configure your .env file with that database name
